scripts/utils/parseBpmn.ts

```diff
@@ -22,7 +22,7 @@ import type { BpmnJsonModel } from '../../src/model/bpmn/json/BPMN20';
 import BpmnXmlParser from '../../src/component/parser/xml/BpmnXmlParser';
 import { newBpmnJsonParser } from '../../src/component/parser/json/BpmnJsonParser';
 import { ParsingMessageCollector } from '../../src/component/parser/parsing-messages';
-import { readFileSync } from '../../test/helpers/file-helper';
+import { readFileSync } from '../../test/shared/file-helper';
```

Comment:
**suggestion:** 
```suggestion
import { readFileSync } from '@test/shared/file-helper';
```

Reply:
❌ Not possible here, it would generate an error `error TS2307: Cannot find module '@test/shared/file-helper' or its corresponding type declarations.`
The path related to @test/shared is only available in the tsconfig file used to build the test code. Here, we use another config file